mne/time_frequency/tests/test_psd.py

```diff
@@ -192,3 +192,10 @@ def test_compares_psd():
     assert (np.sum(freqs_scipy < 0) == 0)
     assert (np.sum(psds_mne < 0) == 0)
     assert (np.sum(psds_scipy < 0) == 0)
+
+
+def test_psd_array_welch_n_jobs():
+    """Test that n_jobs works even with more jobs than channels."""
+    data = np.random.randn(1, 2048)
```

Comment:
```suggestion
    data = np.empty(1, 2048)
```

no need to generate numbers here.